### PR TITLE
Add error counts to CrashAggregator output

### PR DESCRIPTION
**Pull request by maurodoglio**
This is based on top of https://github.com/vitillo/telemetry-streaming/pull/1. Please at the last commit only

### CODE REVIEW

**Comment by maurodoglio** [comment]
RFAL

File: src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala [hunk 1]
```diff
@@ -84,6 +101,10 @@ object ErrorAggregator {
         .asInstanceOf[String])
         .extract[PayloadInfo]
 
+      val keyedHistograms = parse(fields.getOrElse("payload.histograms", "{}")
```
Comment (vitillo) [inline src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala]: Please fix the indentation here.

File: src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala [hunk 2]
```diff
@@ -49,6 +49,13 @@ object ErrorAggregator {
     verify()
   }
 
+  private val countErrorsSchema = new SchemaBuilder()
```
Comment (vitillo) [inline src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala]: Can we call this `countHistogramErrorsSchema`?

**Comment by maurodoglio** [comment]
I'll squash those commits before we merge it

**Comment by vitillo** [comment]
Please rebase and merge :)

**Comment by maurodoglio** [comment]
RFAL

**Comment by vitillo** [comment]
r+

**Pull request merged**